indexing-service/src/main/java/org/apache/druid/indexing/common/task/batch/parallel/ParallelIndexSupervisorTask.java

```diff
@@ -788,18 +785,7 @@ private static void publishSegments(TaskToolbox toolbox, Map<String, PushedSegme
     if (published) {
       LOG.info("Published [%d] segments", newSegments.size());
     } else {
-      LOG.info("Transaction failure while publishing segments, checking if someone else beat us to it.");
```

Comment:
What's the reasoning for removing the more complex logging here? Is it concern from the segment set being too large?

Reply:
Oh, I think the logging here was wrong. In parallel indexing, unlike in streaming ingestion, there is no concept of replicas which could publish the same set of segments. So, if it failed to publish segments, the task should just fail without checking if those segments were already published by another task since it cannot happen.

> "Our segments really do exist, awaiting handoff."

This log is also misleading since batch tasks never waits for segment handoff.